common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala

```diff
@@ -13,14 +14,14 @@ import scala.collection.JavaConversions._
 
 class DynamoTopicSubscriptionsRepository(client: AsyncDynamo, tableName: String)
   (implicit ec: ExecutionContext) extends TopicSubscriptionsRepository {
-
+  val logger = Logger(classOf[DynamoTopicSubscriptionsRepository])
   val TopicIdField = "topicId"
   val TopicSubscriberCountField = "topicSubscriberCount"
 
-  override def subscribe(topic: Topic): Future[RepositoryResult[Unit]] =
+  override def deviceSubscribed(topic: Topic): Future[RepositoryResult[Unit]] =
```

Comment:
Is it necessary to rename these as part of this pull request, or would it be better to make a new PR instead?